pilot/pkg/features/pilot.go

```diff
@@ -524,6 +524,9 @@ var (
 		true,
 		"If false, TCP probes will not be rewritten and therefor always succeed when a sidecar is used.",
 	).Get()
+
```

Comment:
what happens in plain k8s when there are no pods on the same node?

Reply:
> what happens in plain k8s when there are no pods on the same node?

No instance available，will not request instances on other nodes